messagebus/src/main/java/com/yahoo/messagebus/network/NetworkMultiplexer.java

```diff
@@ -68,7 +69,7 @@ else if (owners.contains(owner))
 
     public void unregisterSession(String session, NetworkOwner owner, boolean broadcast) {
         sessions.computeIfPresent(session, (name, owners) -> {
-            if (owners.size() == 1 && owners.contains(owner)) {
+            if (owners.equals(List.of(owner))) {
```

Comment:
Will this equals() really do what's intended? (owners is a `Deque`)

Otherwise LGTM

Reply:
Ah, thanks, that's why it was like it was in the first place!

Reply:
```suggestion
            if (owners.size() == 1 && owners.contains(owner)) {
```